### PR TITLE
OptaPlanner Week videos + blog post

### PR DESCRIPTION
**Pull request by ge0ffrey**

### CODE REVIEW

File: _config/videos.yml [hunk 1]
```diff
@@ -11,6 +11,66 @@
...
+
+- youtubeId: waU8zZtx--g
+  title: "Business use cases and the impact of OptaPlanner"
+  author: satishkale
+  date: 2020-09-03
+  tags:
+    - vehicle routing
+
+- youtubeId: XCyIdmUDMtI
+  title: "Constraint Streams getting started"
```
Comment (triceo) [inline _config/videos.yml:34]:
```suggestion
  title: "Constraint Streams 1-01: The future of score constraints in OptaPlanner"
```

File: _config/videos.yml [hunk 2]
```diff
@@ -11,6 +11,66 @@
...
+  tags:
+    - constraint
+
+- youtubeId: NHU-UZM6fKI
+  title: "Business Optimizer: I bet you I'm better than a human"
+  author: jgoldsmith613
+  date: 2020-09-02
+
+- youtubeId: 07E_v-mZ2As
+  title: "Tasks Optimization - understand the chained models"
```
Comment (triceo) [inline _config/videos.yml:46]:
```suggestion
  title: "Tasks Optimization: understand the chained models"
```

We're using colons everywhere.

File: blog/2020-09-07-OptaPlannerWeek2020.adoc [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+up to 250+ live attendees and 1300+ viewers afterwards.
+We've split up the talks into separate videos for your convenience, watch them below.
+
+Big thanks to all speakers for their superb content,
+all attendees for their great questions during the sessions
+and especially Karina Varela for organizing, hosting and moderating this event perfectly.
+
+image::optaPlannerWeek2020Speakers.jpg[]
+
+The live questions really improved the interactively with the audience.
```
Comment (triceo) [inline blog/2020-09-07-OptaPlannerWeek2020.adoc:19]: Not sure what you're trying to say here. Perhaps `interaction`? `engagement of the audience`?

File: blog/2020-09-07-OptaPlannerWeek2020.adoc [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+video::E35pYMKaukw[youtube]
+
+=== Planning agility: continuous planning, real-time planning and more
+video::Ew6pq9nJKog[youtube]
+
+
+== Next year
+
+See you next year!
```
Comment (triceo) [inline blog/2020-09-07-OptaPlannerWeek2020.adoc:78]: 1. Don't make commitments.
1. If you must, at least don't waste 2 lines on something where `See you next year!` will be just fine.

File: blog/2020-09-07-OptaPlannerWeek2020.adoc [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+video::MZgWrKwfVx0[youtube]
+
+=== TrustyAI and explanation by example with OptaPlanner
+
+video::4H3U6xyCgMI[youtube]
+
+
+== Day 2
+
+=== Tasks Optimization - understand the chained models
```
Comment (triceo) [inline blog/2020-09-07-OptaPlannerWeek2020.adoc:49]:
```suggestion
=== Tasks Optimization: understand the chained models
```

File: blog/2020-09-07-OptaPlannerWeek2020.adoc [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+video::07E_v-mZ2As[youtube]
+
+=== Business optimizer: I bet you I'm better than a human
+
+video::NHU-UZM6fKI[youtube]
+
+=== Constraint streams getting started
```
Comment (triceo) [inline blog/2020-09-07-OptaPlannerWeek2020.adoc:57]:
```suggestion
=== Constraint Streams 1-01: The future of score constraints in OptaPlanner
```

**Review by triceo — APPROVED** [review]
Suggesting tweaks here and there.

File: blog/2020-09-07-OptaPlannerWeek2020.adoc (hunk 4 above)
Comment (ge0ffrey) [reply blog/2020-09-07-OptaPlannerWeek2020.adoc:78]: For 2: I don't want this paragraph to be part of the section of the last talk

Comment (ge0ffrey) [reply blog/2020-09-07-OptaPlannerWeek2020.adoc:78]: for 1: it was said several times in the finishing fire-side chat, I think the cat is out of the box :)
And it's a good thing!

**Pull request merged**